deploying/rhoaiserving-using/validate_model_servers.ipynb

```diff
@@ -0,0 +1,114 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip -qq install transformers\n",
```

Comment:
Pin the transfomers version or we might run into problems with future releases.

```suggestion
    "!pip -qq install transformers==4.46.3\n",
```

Reply:
Or we can include a requirements file and then run something like `pip install -r requirements.txt`